src/types/customerInstitution.types.ts

```diff
@@ -32,6 +32,7 @@ export interface CustomerInstitution extends Pick<SimpleCustomerInstitution, 'id
   rorId?: string;
   doiAgent: DoiAgent;
   sector: Sector;
+  inactiveFrom?: string | null;
```

Comment:
denne er `undefined` om aldri satt, `string` om dato, `null` om den har hatt en dato som i ettertid er fjernet?
Tom streng er kanskje det samme som `null`?

Spør bare for å finne ut om vi trenger alle, da `undefined`  og `string` kan se ut til å dekke alle tilfeller for meg

Reply:
og det blir 400-error om klient sender inn noe som ikke er en Date?

Reply:
Ja blir error hvis klienten sender inn noe som ikke er en dato, serveren konverterer strengen til java instant, så hvis den ikke parses så hvil man få noe feil.

Reply:
Endrer til inactiveFrom?: string